howto/track-rethinkdb-state-store-changes/README.md

```diff
@@ -0,0 +1,26 @@
+# How to track RethinkDB state store changes 
+
+The RethinkDB state store supports transactions which means it can be used to support Dapr actors. Dapr persisting only the actor's current state which doesn't allow the users to track how actor's state may have changed over time.
```

Comment:
```suggestion
The RethinkDB state store supports transactions which means it can be used to support Dapr actors. Dapr persists only the actor's current state which doesn't allow the users to track how actor's state may have changed over time.
```